pkg/apis/dynatrace/v1alpha1/oneagent_types.go

```diff
@@ -45,6 +45,8 @@ type OneAgentSpec struct {
 	EnableIstio bool `json:"enableIstio,omitempty"`
 	// DNS Policy for the OneAgent pods.
 	DNSPolicy corev1.DNSPolicy `json:"dnsPolicy,omitempty"`
+	// Labels for the OneAgent pods
+	Labels map[string]string `json:"labels, omitempty"`
```

Comment:
Hmm, I think the whitespace before omitempty marking this field as required on the CRD above.